static/components/AroundCards.js

```diff
@@ -32,16 +33,28 @@ class App extends React.Component {
       })
     })
     if (currentLocation.length == 0) {
-      const localPosition = localStorage.userPosition.split(',', 2)
+        navigator.geolocation.getCurrentPosition(position => {
+          userPosition = [position.coords.longitude, position.coords.latitude]
+          focusUser(userPosition)
+          this.setState({
+            userPosition: [userPosition[1], userPosition[0]]
```

Comment:
seems to be reversed compared to what was used in the previous commit.
Not sure if you're going for latLng or lngLat ([standards being latlng](http://stackoverflow.com/questions/7309121/preferred-order-of-writing-latitude-longitude-tuples?answertab=votes#tab-top))